src/Core/Utils/Color.hpp

```diff
@@ -32,7 +32,7 @@ class ColorBase : public Eigen::Matrix<_Scalar, 4, 1>
 
     /// Copy constructor from Eigen expressions: `ColorBase<_Scalar> c( (*this) * 255 );`
     template <typename Derived>
-    explicit inline ColorBase( const Eigen::MatrixBase<Derived>& v ) :
+    inline ColorBase( const Eigen::MatrixBase<Derived>& v ) :
```

Comment:
I would have used // cppcheck-suppress noExplicitConstructor but thereis no (simple) way to configure codacy so that cppcheck is run with the `--inline-suppr`.